editor/components/rich-text/index.js

```diff
@@ -439,6 +442,18 @@ export class RichText extends Component {
 		};
 	}
 
+	getInsertPosition() {
```

Comment:
Why is this not part of `InlineBlocks` if the `editor` instance is passed?

Reply:
I had left if there since the `containerRef` and the similar `getFocusPosition` function were in `RichText`, but I agree that it belongs with `InlineBlocks` now, and moved it in https://github.com/WordPress/gutenberg/pull/5794/commits/b8a225acf9479b2dc8ae18cb73feab5168b62d0c.